packages/admin/admin/src/mui/menu/Item.styles.ts

```diff
@@ -29,6 +29,11 @@ export const styles = (theme: Theme) => {
             "& [class*='MuiListItemText-inset']": {
                 paddingLeft: ({ icon, level }) => (!!icon && level === 1 ? 28 : 0),
             },
+            "&[class*='Mui-selected']": {
+                "& [class*='MuiListItemText-secondary']": {
+                    color: theme.palette.primary.dark,
+                },
+            },
```

Comment:
Should this maybe just always be "inherit" so it's always the same color, as the primary text?

```suggestion
            "& [class*='MuiListItemText-secondary']": {
                color: "inherit",
            },
```

Reply:
Or maybe just on Selected, if it should not be black by default 🤔 

```suggestion
            "&[class*='Mui-selected']": {
                "& [class*='MuiListItemText-secondary']": {
                    color: "inherit",
                },
            },
```

Reply:
Shouldn't it differ as it differs for the unselected version?

Reply:
Do you mean it should be darker than the primary Text? 
I think it makes sense to be the same color when selected but I don't know how the UX-Team intended this, I couldn't find this case in the designs.

Reply:
I'll ask

Reply:
changed it to inherit following the updated design